samples/java/src/main/java/com/gradle/enterprise/export/ExportApiJavaExample.java

```diff
@@ -66,7 +70,8 @@ public static void main(String[] args) throws Exception {
         AtomicReference<String> eventId = new AtomicReference<>();
 
         HttpClientRequest<ByteBuf, ByteBuf> request = HTTP_CLIENT
-            .createGet(url);
+            .createGet(url)
+            .addHeader("Authorization", "Basic " + Base64.getEncoder().encodeToString(EXPORT_API_CREDENTIALS.getBytes()));
```

Comment:
Can you do this `Base64` encoding to where we define the `EXPORT_API_CREDENTIALS` variable such that we only do this once per run and not every time?

Reply:
Done.